flytekit/clis/flyte_cli/main.py

```diff
@@ -2359,26 +2367,27 @@ def setup_config(host, insecure):
     config_url = _urlparse.urljoin(full_host, "config/v1/flyte_client")
     response = _requests.get(config_url)
     data = response.json()
+    platform_config = {"url": str(host), "insecure": str(insecure)}
+    credentials_config = None
+    if not insecure:
+        # We can't get credentials config in insecure mode
+        credentials_config = {
+            "client_id": data["client_id"],
+            "redirect_uri": data["redirect_uri"],
+            "scopes": data["scopes"],
+            "authorization_metadata_key": data["authorization_metadata_key"],
+            "auth_mode": "standard",
+        }
     with open(config_file, "w+") as f:
-        f.write("[platform]")
-        f.write("\n")
-        f.write("url={}".format(host))
-        f.write("\n")
-        f.write("insecure={}".format(insecure))
-        f.write("\n\n")
-
-        f.write("[credentials]")
-        f.write("\n")
-        f.write("client_id={}".format(data["client_id"]))
```

Comment:
cc @wild-endeavor. 
```
File "/Users/ytong/envs/tstsetup/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/ytong/envs/tstsetup/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/ytong/envs/tstsetup/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Users/ytong/envs/tstsetup/lib/python3.8/site-packages/flytekit/clis/flyte_cli/main.py", line 2372, in setup_config
    f.write("client_id={}".format(data["client_id"]))
KeyError: 'client_id'
```
It seems like you used the old version of flytekit.
I already remove `f.write`